xrpl/models/transactions/nftoken_burn.py

```diff
@@ -39,6 +40,15 @@ class NFTokenBurn(Transaction):
     :meta hide-value:
     """
 
+    owner: Optional[str] = None
+    """
+    Indicates which account currently owns the token if it is different than
+    Account. Only used to burn tokens which have the lsfBurnable flag enabled
+    and are not owned by the signing account.
+
+    :meta hide-value:
```

Comment:
The value of this shouldn't be hidden - that's only used for ones where the default value is `REQUIRED`